app/models.py

```diff
@@ -437,6 +465,21 @@ class UserEvent(Event):
     def from_user(cls, user, **kwargs):
         return cls(user_id=user.id, deployment=user.deployment, **kwargs)
 
+class SimpleUserEvent(UserEvent):
+    '''
+    Any kind of user event that doesn't have any extra data associated
+    with it.
+    '''
```

Comment:
This *seems* like a reasonable thing to do based on the mockups, where we have a number of simple events that don't have any extra metadata associated with them, like "A user joined the network" and "A user added expertise".